docs/diagrams/GroupState1.puml

```diff
@@ -0,0 +1,23 @@
+@startuml
+'https://plantuml.com/class-diagram
+
+hide circle
+hide methods
+
+skinparam ClassFontColor #000000
+skinparam ClassBorderColor #000000
+skinparam ClassBackgroundColor #FFFFAA
+
+title Current state before modification
+
+package addressBook <<rectangle>> {
+    class State1 as "<u>p0:Person</u>" {
+        studentId = A0123456X
```

Comment:
Looks good. Do you think it would be appropriate to show `StudentId` as an object too since each `Person` and `StudentId` object actually has a whole-part relationship if you refer to the ModelClassDiagram.
![ModelCLassDiagram](https://github.com/AY2324S2-CS2103T-T09-1/tp/assets/108446221/5db389b2-8e98-46ef-bd3e-0477ce45d1ca)

Reply:
Thanks for the suggestion. I didn't do so initially because I would just be drawing an additional box to show the same information. So it might end up more cluttered. However, given that several other diagrams are already written in that manner I will make the suggested changes to standardize the documentation. I won't show the irrelevant fields like grade, email etc. though.